hoomd/conftest.py

```diff
@@ -196,6 +220,9 @@ def pytest_configure(config):
     config.addinivalue_line(
         "markers",
         "cupy_optional: tests that should pass with and without CuPy.")
+    config.addinivalue_line(
+        "markers",
+        "validate: Tests that perform longer validation tests.")
```

Comment:
Slightly less redundant phrasing:
```suggestion
        "validate: Tests that perform long-running validations.")
```